docs/cody/model-configuration/examples.mdx

```diff
@@ -0,0 +1,681 @@
+| `accessToken` | Leave empty to rely on instance role bindings or other AWS configurations in the frontend service. Use `<ACCESS_KEY_ID>:<SECRET_ACCESS_KEY>` for direct credential configuration, or `<ACCESS_KEY_ID>:<SECRET_ACCESS_KEY>:<SESSION_TOKEN>` if a session token is also required.                                 |
+| `endpoint`    | For Pay-as-you-go, set it to an AWS region code (e.g., `us-west-2`) when using a public Amazon Bedrock endpoint. For Provisioned Throughput, set it to the provisioned VPC endpoint for the bedrock-runtime API (e.g., `https://vpce-0a10b2345cd67e89f-abc0defg.bedrock-runtime.us-west-2.vpce.amazonaws.com`). |
+| `region`      | The region to use when configuring API clients. This is necessary because the 'frontend' binary's container cannot access environment variables from the host OS.                                                                                                                                               |
+
+Provisioned throughput for AWS Bedrock models can be configured using the `"awsBedrockProvisionedThroughput"` server-side
+configuration type. For more details, refer to the [Model Overrides](/cody/model-configuration#model-overrides) section.
+
+</Accordion>
+
+## Self-hosted models
+
+<Callout type="note">
+	Configuring self-hosted models can be nuanced and complex. Consider reaching
+	out to Sourcegraph support for assistance with the setup.
+</Callout>
+
+For detailed information about configuring self-hosted models, please refer to the [self-hosted models](/cody/model-configuration#self-hosted-models) documentation.
+
+Example configuration:
+
+```json
+"cody.enabled": true,
+"modelConfiguration": {
+  "sourcegraph": null,
+  "providerOverrides": [
+    {
+      "id": "fireworks",
+      "displayName": "Fireworks",
+      "serverSideConfig": {
+        "type": "openaicompatible",
+        "endpoints": [
+          {
+            "url": "https://api.fireworks.ai/inference/v1",
+            "accessToken": "token"
+          }
+        ]
+      }
+    }
+  ],
+  "selfHostedModels": [
+    {
+      "provider": "fireworks",
+      "model": "mixtral-8x7b-instruct@v1",
+      "override": {
+        "displayName": "(Fireworks) Mixtral 8x7b Instruct",
+        "serverSideConfig": {
+          "type": "openaicompatible",
+          "apiModel": "accounts/fireworks/models/mixtral-8x7b-instruct"
+        }
+      }
+    },
+    {
+      "provider": "fireworks",
+      "model": "mixtral-8x22b-instruct@v1",
+      "override": {
+        "displayName": "(Fireworks) Mixtral 8x22b Instruct",
+        "serverSideConfig": {
+          "type": "openaicompatible",
+          "apiModel": "accounts/fireworks/models/mixtral-8x22b-instruct"
+        }
+      }
+    },
+  ]
+}
+```
+
+In the configuration above we:
+
+-   TODO
```

Comment:
@slimsag, can you please help with the self-hosted models example description?

Reply:
I will return to this on Monday.

Reply:
@taras-yemets it's still a To Do

Reply:
@MaedahBatool, I didn't get to it today. On top of my list for tomorrow.